core/src/test/java/org/apache/calcite/test/SqlToRelTestBase.java

```diff
@@ -529,6 +553,45 @@ public RelRoot convertSqlToRel(String sql) {
       return root;
     }
 
+    public RelRoot convertSqlToRel(String sql, SqlToRelConverter.Config config) {
```

Comment:
I think this can follow the similar way as the SqlToRelConverter constructor.  You do not have to duplicate code from convertSqlToRel(String sql). Just pass the default config instance in convertSqlToRel(String sql).